larray/core.py

```diff
@@ -9265,8 +9265,8 @@ def read_csv(filepath, nb_index=None, index_col=None, sep=',', headersep=None, n
 
     Parameters
     ----------
-    filepath : str
-        Path where the csv file has to be read.
+    filepath : str or any object with a read() method
+        Path where the csv file has to be read or a file handle or a StringIO object.
```

Comment:
unsure we should explicitly mention StringIO, because that is just one kind of file handle, right?